src/module/gui/__init__.py

```diff
@@ -139,6 +140,10 @@ def set_icon(self, img: QImage|None):
 			self.icon.fill(QColor(0, 0, 0, 0))
 			self.iconButton.setIcon(self.icon)
 		print('Icon updated!')
+		if self.path is not None:
+			self.iconButton.setToolTip(f'{os.path.basename(self.path)}  Right-click to remove')
```

Comment:
Since this is already a Path, we can just use `self.path.name` instead of `os.path.basename(self.path)`